client/src/components/Dashboard/index.tsx

```diff
@@ -254,6 +263,11 @@ export default class Dashboard extends React.Component<IDashboardProps, IDashboa
     // Actions to perform on an active dashboard
     let toolbarActions = [
       (
+      <span><Button key="downloadDashboard" icon tooltipLabel="Download Dashboard" onClick={this.onDownloadDashboard}>
```

Comment:
This is not a must:
I think it makes sense for this functionality to be put inside a different React component